i18n/fi.yaml

```diff
@@ -0,0 +1,116 @@
+- id: home
+  translation: "Koti"
+
+- id: templateBy
+  translation: "Sivun pohjan kehitti"
+
+- id: portedBy
+  translation: "Portannut Hugolle"
+
+- id: contactGoTo
+  translation: "Siirry yhteydenottolomakkeeseen"
+
+- id: contactAddrTitle
+  translation: "Osoite"
+
+- id: contactTitle
+  translation: "Ota yhteyttä"
+
+- id: contactForm
+  translation: "Yhteydenottolomake"
+
+- id: contactName
+  translation: "Nimi"
+
+- id: contactMail
+  translation: "Sähköposti"
+
+- id: contactMessage
+  translation: "Viesti"
+
+- id: contactSend
+  translation: "Lähetä"
+
+- id: navHome
+  translation: "siirry kotisivulle"
```

Comment:
Letter casing here inconsistent

Reply:
Same thing with the translations context in here. How it's defined in `themes/hugo-universal-theme/i18n/en.yaml`:
```yaml
- id: navHome
  translation: "go to homepage"
```
and how the translation is used in `themes/hugo-universal-theme/layouts/partials/nav.html`:
```html
<span class="sr-only">{{ .Title }} - {{ i18n "navHome" }}</span>
```

Reply:
Alright I see, thanks for taking the time to point that out! 😊

Reply:
It's all good 🙂